addons/project/static/src/views/project_form/project_settings.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { Setting } from "@web/views/form/setting/setting";
+
+export class ProjectSettings extends Setting {
+    /**
+     * Replace `col-lg-6` with `col-lg-12'.
+     *
+     * @override method from Setting componet
+     * @return {object}
+     */
+    get classNames() {
+        const { class: _class } = this.props;
+        return {
+            o_setting_box: true,
+            "col-12": true,
+            "col-lg-12": true,
+            [_class]: Boolean(_class),
+        };
+    }
```

Comment:
I am not sure to understand how we replace `col-lg-6` by `col-lg-12` with this code.

Also, why do you not call super method? 🤔 

I am wondering if we should not replace the class set in the xml view instead of doing a customisation like that. 

What do you think?

Reply:
> I am not sure to understand how we replace `col-lg-6` by `col-lg-12` with this code.
> 
> Also, why do you not call super method? thinking
> 
> I am wondering if we should not replace the class set in the xml view instead of doing a customisation like that.
> 
> What do you think?

It's possible through xml. we have to added class `col-lg-6` in  all setting tags. So I think we should override the setting component. We do  through xml in the help desk team. [helpdesk team](https://github.com/odoo-dev/enterprise/commit/55043d52280895aef7ce3f3975a080625c8c5888#diff-2c23f3a88874a7461ba99f08abef566f792f029c82b4dfe7ff56fd900b8592c3R100)

`why do you not call super method ?`: super method returns 'col-lg-6' class we not need that class.so we do not call super. [super](https://github.com/odoo/odoo/blob/master/addons/web/static/src/views/form/setting/setting.js#L21C2-L31C6).

what do you think now?